documentation/src/main/wiki/Dictionary.md

```diff
@@ -63,19 +73,28 @@ a module responsible for filtering the collected data before performing comparis
 *Data Storage*  
 a database abstraction layer which contains versioned data (data grid).
 
+*Execute Java Script Modifier*  
+a modifier that allows to execute JavaScript code.
```

Comment:
We could add:  _on tested page._ to this sentence, what do you think?

Reply:
Good idea!

Reply:
Done.